cpplint_unittest.py

```diff
@@ -320,8 +342,16 @@ def testFalsePositivesNoError(self):
   # Test get line width.
   def testGetLineWidth(self):
     self.assertEquals(0, cpplint.GetLineWidth(''))
+<<<<<<< b2e46ca571a1cff5431a12a5cac8e4fc856ce852
     self.assertEquals(10, cpplint.GetLineWidth('x' * 10))
     self.assertEquals(16, cpplint.GetLineWidth(u('\u90fd|\u9053|\u5e9c|\u770c|\u652f\u5e81')))
+=======
+    self.assertEquals(10, cpplint.GetLineWidth(unicode('x') * 10))
+    try:
+      self.assertEquals(16, cpplint.GetLineWidth('都|道|府|県|支庁'.decode('utf-8')))
+    except AttributeError:
+      self.assertEquals(16, cpplint.GetLineWidth('都|道|府|県|支庁'))
+>>>>>>> Added support for Python3000
```

Comment:
* you convert 'x' to unicode, I don't
* you check with AttributeError, I use function u() with unicode notation